ci/travis/check-git-clang-format-output.sh

```diff
@@ -8,7 +8,7 @@ else
   base_commit="$TRAVIS_BRANCH"
   echo "Running clang-format against branch $base_commit, with hash $(git rev-parse $base_commit)"
 fi
-output="$(ci/travis/git-clang-format --binary clang-format-3.8 --commit $base_commit --diff --exclude '(.*thirdparty/|.*redismodule.h|.*.js|.*.java)')"
+output="$(ci/travis/git-clang-format --binary clang-format --commit $base_commit --diff --exclude '(.*thirdparty/|.*redismodule.h|.*.js|.*.java)')"
```

Comment:
Could be done in a separate PR, but we should throw an error if the version of `clang-format` is not the one we test in the CI.

Same with `flake8` and `yapf`, we should raise exceptions if the user has a different version (in the `ci/travis/format.sh` script).